src/components/exercises/App1/redux/slices/contactSlice.js

```diff
@@ -10,6 +10,20 @@ export const fetchContacts = createAsyncThunk(
     return response.contacts;
   }
 );
+export const deleteContactThunk = createAsyncThunk(
+  "contact/contactID",
+  async (data) => {
+    const response = await fetch("delete-contact", {
```

Comment:
The API URL should be DELETE contact/:contactId.
Method should not be POST but DELETE